script.js

```diff
@@ -0,0 +1,66 @@
+let word = 'APPLE'
+
+// Hide the word: 
+document.getElementById('word').innerHTML = word
+document.getElementById('word').style.display = 'none'
+
+// Check how many letters and write the _ :
+let underscore = word.length
+
+switch (underscore){
+    case '1':
```

Comment:
Then you can use this array to generate the amount of underscores by using a loop
Let's say you create a loop and for each letter in the array you return an underscore
For example, for each letter, you can change the innerHTML of some element by adding to it:

underscoreHTML += `<div class="underscore">_</div>`;

I think I have seen you do something similar in a project before 💡
And if you need the class name to be different for each underscore, you can use the index number in the for loop and add it to the class name.
This was all underscores can have a different class name like: "underscore-0", "underscore-1", etc...

Reply:
I think I didn't understand this part